openassessment/xblock/static/sass/xb/elements/_controls.scss

```diff
@@ -108,7 +108,6 @@
 
 %btn-reset {
```

Comment:
I am worried about potential fallout. Can you style just the things that aren't really buttons with the size 14?